### PR TITLE
Fix the page headers!

### PR DESCRIPTION
**Pull request by ryelo**
Updating the page headers to use correct spacing.

Jiras:
* https://projects.engineering.redhat.com/browse/RHCLOUD-4346
* https://projects.engineering.redhat.com/browse/RHCLOUD-4446

### Regular page header
* Purple is 24px
Screen Shot 2020-02-17 at 3 29 57 PM

### Page header with description
* Purple is 24px
Screen Shot 2020-02-17 at 3 51 58 PM

### Page header with tabs, description, breadcrumbs
* Purple is 24px
* Yellow is 8px
Screen Shot 2020-02-17 at 3 27 32 PM

Screen Shot 2020-02-17 at 3 27 44 PM

### No markers
Screen Shot 2020-02-17 at 3 27 13 PM

### CODE REVIEW

**Comment by ryelo** [comment]
cc: @mmenestr

**Comment by mmenestr** [comment]
image

File: src/presentational-components/shared/top-toolbar.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
-import { Level, LevelItem, Text, TextContent, TextVariants  } from '@patternfly/react-core';
-import clsx from 'clsx';
+import { Text, TextContent, TextVariants  } from '@patternfly/react-core';
 import { ToolbarTitlePlaceholder } from './loader-placeholders';
 import RbacBreadcrumbs from './breadcrubms';
+import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
```
Comment (karelhala) [inline src/presentational-components/shared/top-toolbar.js]: Can you use direct imports? So we can start trimming the bundle size. `import { PageHeader, PageHeaderTitle } from 
 '@redhat-cloud-services/frontend-components/components/PageHeader'`

File: src/test/role/__snapshots__/role.test.js.snap [hunk 2]
```diff
@@ -609,16 +609,4 @@ Array [
 ]
 `;
 
-exports[`role should render top toolbar 1`] = `
```
Comment (karelhala) [inline src/test/role/__snapshots__/role.test.js.snap]: Looks like this test is broken, can you change in `role.test.js` finding of `.top-toolbar` to `expect(toJson(wrapper.find('TopToolbar'), { mode: 'shallow' })).toMatchSnapshot();`

**Pull request merged**